### DISCUSSION TITLE
Get accesstoken failed. Username or password error!

### DISCUSSION THREAD

**Discussion by 5Fishes** [milo526/homebridge-tuya-web#567] [Q&A]
Hi @milo526 - I tested your python script [here](https://github.com/milo526/homebridge-tuya-web/tree/master/tools) and I get the same error as when I run the plug-in for Homebridge.
ERROR:
python3 TuyaWebCheck.py
Getting credentials
Got credentials
{   'errorMsg': 'Get accesstoken failed. Username or password error!',
    'responseStatus': 'error'}
Traceback (most recent call last):
  File "/home/runner/TestTuyaWeb/main.py", line 35, in <module>
    payload = {"accessToken": auth_response["access_token"]}
KeyError: 'access_token'
I checked the common things you mentioned:
Are you using sign-in with Apple/Facebook/Google/Whatever? - NO
Did you set the correct country-code? - YES
Did you set the correct platform? - YES
Are there any dots in your email (you already said this is not the case). - NO
Thoughts?

**Comment by mlouism** [comment]
I  saw somewhere that is a "dot" issue on user mail password. To fix that,  it has proposal to using mobile number to reach site like iot.tuya.com, the problem is according to Tuya support, you have to live in the good part of the world i.e. Asia  or somewhere between because they said they don't support in the other part of the world to login with mobile number! (like America!!!)
